tutorials/names.json

```diff
@@ -1,72 +1,75 @@
-            "CreatingLanguages": {
-                "title": "Creating Languages"
+	"GettingStarted": {
+		"title": "Getting Started",
+		"children": {
+			"CreatingCommands": {
+				"title": "Creating Commands"
+			},
+			"CreatingEvents": {
+				"title": "Creating Events"
+			},
+			"CreatingExtendables": {
+				"title": "Creating Extendables"
+			},
+			"CreatingFinalizers": {
+				"title": "Creating Finalizers"
 			},
-            "CreatingMonitors": {
-                "title": "Creating Monitors"
-            },
-            "CreatingProviders": {
-                "title": "Creating Providers"
-            }
-        }
-    },
-    "UnderstandingUsageStrings": {
-        "title": "Understanding Usage Strings"
+			"CreatingInhibitors": {
+				"title": "Creating Inhibitors"
+			},
+			"CreatingLanguages": {
+				"title": "Creating Languages"
+			},
+			"CreatingMonitors": {
+				"title": "Creating Monitors"
+			},
+			"CreatingProviders": {
+				"title": "Creating Providers"
+			}
+		}
+	},
+	"UnderstandingUsageStrings": {
+		"title": "Understanding Usage Strings"
 	},
-    "UnderstandingPermissionLevels": {
-        "title": "Understanding Permission Levels"
+	"UnderstandingPermissionLevels": {
+		"title": "Understanding Permission Levels"
 	},
-    "UnderstandingSettingGateway": {
-        "title": "Understanding Setting Gateway"
+	"UnderstandingSettingGateway": {
+		"title": "Understanding Setting Gateway"
 	},
 	"IncludedPieces": {
 		"title": "Included Pieces",
 		"children": {
-            "IncludedCommands": {
-                "title": "Included Commands"
-            },
-            "IncludedEvents": {
-                "title": "Included Events"
-            },
-            "IncludedExtendables": {
-                "title": "Included Extendables"
-            },
-            "IncludedFinalizers": {
-                "title": "Included Finalizers"
-            },
-            "IncludedInhibitors": {
-                "title": "Included Inhibitors"
-            },
-            "IncludedLanguages": {
-                "title": "Included Languages"
+			"IncludedCommands": {
+				"title": "Included Commands"
 			},
-            "IncludedMonitors": {
-                "title": "Included Monitors"
-            },
-            "IncludedProviders": {
-                "title": "Included Providers"
-            }
-        }
+			"IncludedEvents": {
+				"title": "Included Events"
+			},
+			"IncludedExtendables": {
+				"title": "Included Extendables"
+			},
+			"IncludedFinalizers": {
+				"title": "Included Finalizers"
+			},
+			"IncludedInhibitors": {
+				"title": "Included Inhibitors"
+			},
+			"IncludedLanguages": {
+				"title": "Included Languages"
+			},
+			"IncludedMonitors": {
+				"title": "Included Monitors"
+			},
+			"IncludedProviders": {
+				"title": "Included Providers"
+			}
+		}
 	},
 	"PieceStores": {
 		"title": "Custom Pieces and Stores"
+	},
+	"RichDisplay": {
+		"title": "Using RichDisplay for paginated embeds"
```

Comment:
Should use proper title capitalization

Reply:
Grammatically, it is properly capitalized

Reply:
`Using RichDisplay for Paginated Embeds` looks better

Reply:
__The capitalization rules are as follow:__
1. Capitalize words with 4 or more letters.
2. Capitalize the first and the last word.
3. Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
4. Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.

paginated is an adjective with more than 4 letters, thus rules 1 & 3 applies.
embeds is both the last word and a noun with more than 4 letters, thus rules 1 & 2 & 3 apply